qemu/tests/rng_hotplug.py

```diff
@@ -81,14 +85,16 @@ def stop_rngd(vm):
     rng_num = int(params.get("rng_num", 1))
     test_before_hotplug = params.get("test_before_hotplug")
     test_after_hotplug = params.get("test_after_hotplug")
+    sub_type_after_plug = params.get("sub_type_after_plug")
+    sub_type_after_unplug = params.get("sub_type_after_unplug")
 
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     vm.wait_for_login(timeout=login_timeout)
 
     if test_before_hotplug:
         restart_rngd(vm)
-        error_context.context("Run %s before hotplug" % test_before_hotplug)
+        error_context.context("Run %s before hot-unplug" % test_before_hotplug)
```

Comment:
It is confusing that log shows "before hot-unplug" while the variant is named "test_before_hotplug".

Reply:
I change from "hotplug" to "unplug" just to maker testers know that this is a check step before hot-unplug action.

the original variant "test_before_hotplug" actually means test_before_hotunplug,I can change it as well.